index.html

```diff
@@ -11,6 +11,7 @@
       <base href="/">
       <!-- @if ENV='dev' -->
       <link rel="stylesheet" href="dist/give.css" type="text/css">
+      <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
```

Comment:
@adammeyer , inside the modal/bank account there is a question mark at the bottom.